config/environments/production.rb

```diff
@@ -69,7 +69,13 @@
   config.i18n.fallbacks = true
 
   config.paperclip_defaults = {
-      storage: :file
+      storage: :s3,
+      s3_host_name: 's3-us-west-2.amazonaws.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.